data/remnant/remnant jobs.txt

```diff
@@ -659,3 +659,37 @@ mission "Remnant: Rescue 5"
 	on visit
 		dialog
 			`You have returned, but there is still no word from the missing ship.`
+
+
+mission "Remnant: Disable 1"
+	job
+	repeat
+	name "Disable Damaged Kas'lor Ik 582"
+	description "A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Will Not Someone Please Think Of The Children: done"
+		random < 5
+	npc disable save
+		government "Korath"
+		personality heroic vindictive target marked staying
+		system 
+			near "Antevorta" 1
+		fleet
+			names "korath"
+			variant
+				"Kas'lor Ik 582 (Crippled)"
```

Comment:
Just thinking, we could maybe have another variant? 
Should prob be done in another PR after this one is merged though. I'm thinking a 577 or a 590?

Reply:
582 was selected for these reasons outlined by @ravenshining 
https://github.com/endless-sky/endless-sky/pull/7689#pullrequestreview-1193000780

TLDR Effectively of the civilian ships only 582 should be seen outside the Korath core sectors. And then it should be assumed that it is on some type of refueling mission.